<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vastMainWindow</class>
 <widget class="QMainWindow" name="vastMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vast</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>762</width>
         <height>513</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="topScrollAreaGridLayout">
          <property name="horizontalSpacing">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="widgetGridLayout">
          <property name="horizontalSpacing">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="bottomScrollAreaGridLayout">
          <property name="horizontalSpacing">
           <number>0</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QGridLayout" name="bottomGridLayout">
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="topGridLayout"/>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="leftGridLayout"/>
    </item>
    <item row="1" column="2">
     <layout class="QGridLayout" name="rightGridLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open_image"/>
    <addaction name="actionOpen_directory"/>
    <addaction name="actionOpen_Zmap"/>
    <addaction name="separator"/>
    <addaction name="action_Save_Image"/>
    <addaction name="actionSave_Image"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Open_image">
   <property name="text">
    <string>&amp;Open Image</string>
   </property>
   <property name="shortcut">
    <string>O, I</string>
   </property>
  </action>
  <action name="actionOpen_directory">
   <property name="text">
    <string>Open &amp;Directory</string>
   </property>
   <property name="shortcut">
    <string>O, D</string>
   </property>
  </action>
  <action name="actionOpen_Zmap">
   <property name="text">
    <string>Open &amp;Zmap</string>
   </property>
   <property name="shortcut">
    <string>O, Z</string>
   </property>
  </action>
  <action name="action_Save_Image">
   <property name="text">
    <string>&amp;Save Image</string>
   </property>
   <property name="shortcut">
    <string>S, I</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="text">
    <string>Save Image...</string>
   </property>
   <property name="shortcut">
    <string>S, I, A</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
